packages/xo-server/src/api/schedule.mjs

```diff
@@ -64,3 +66,26 @@ delete_.params = {
 }
 
 export { delete_ as delete }
+
+export async function runSequence({ idSchedules }) {
+  const t = new Task({ properties: { type: 'xo:schedule:sequence', name: 'Schedule sequence' } })
+  await t.run(async () => {
+    const nb = idSchedules.length
+    const signal = Task.abortSignal
+    for (let i = 0; i < nb; i++) {
+      if (signal.aborted) {
+        throw new Error(signal.reason)
+      }
```

Comment:
It is better to use `throwIfAborted()`, it does not wrap the reason in an `Error` object because it can already be ones (which is the default when `abortController.abort()` without explicit reason).

```suggestion
      signal.throwIfAborted()
```